src/modules/commander/accelerometer_calibration.cpp

```diff
@@ -469,7 +469,7 @@ calibrate_return do_accel_calibration_measurements(orb_advert_t *mavlink_log_pub
 
 	// Warn that we will not calibrate more than max_accels accelerometers
 	if (orb_accel_count > max_accel_sens) {
-		calibration_log_critical(mavlink_log_pub, "[cal] Detected %u accels, but will calibrate only %u", orb_accel_count, max_accel_sens);
+        calibration_log_critical(mavlink_log_pub, "Detected %u accels, but will calibrate only %u", orb_accel_count, max_accel_sens);
```

Comment:
Firmware uses tabs and not spaces. Depending on your editor setup it can be hard to see. This is one of the few tools where it isn't enforced (yet). For most of the rest of the codebase you can fix it automatically with astyle then run `make check_format` or `make format` locally.

Reply:
I'm using Qt as my IDE.  Is there a tutorial or spec sheet for me to update my code formatting config in the IDE?